canvas_sdk/effects/banner_alert/remove_banner_alert.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from canvas_sdk.effects.base import _BaseEffect
+
+
+class RemoveBannerAlert(_BaseEffect):
+    """
+    An Effect that will remove/inactivate a banner alert in Canvas.
+    """
+
+    class Meta:
+        effect_type = "REMOVE_BANNER_ALERT"
```

Comment:
Does this need to be the constant `EffectType.REMOVE_BANNER_ALERT`? I'm not sure how this works as a string.